src/components/apps/details/DetailsPanel.js

```diff
@@ -105,7 +105,11 @@ function DetailsPanel(props) {
                             </GridLabelValue>
                         )}
                         <GridLabelValue label={t("publishedOn")}>
-                            {formatDate(details.integration_date)}
+                            {details.integration_date
+                                ? formatDateObject(
+                                      new Date(details.integration_date)
+                                  )
+                                : "-"}
```

Comment:
`formatDateObject` already has `-` as a constant for returning an empty date string.
```suggestion
                            {formatDateObject(
                                details.integration_date && new Date(details.integration_date)
                            )}
```